docs/source/support/faq.rst

```diff
@@ -0,0 +1,168 @@
+==========================
+Frequently Asked Questions
+==========================
+
+What are the recommended specifications for running a nucypher node?
+--------------------------------------------------------------------
+
+Worker nodes need to run ``nucypher`` and a local ethereum node. In total, you will
+require at 4GB for RAM. Nodes also need 24/7 uptime and a static, public IPv4 address.
+
+For ``nucypher`` specific requirements, see `System Requirements and Dependencies <https://docs.nucypher.com/en/latest/guides/installation_guide.html#system-requirements-and-dependencies/>`_.
+
+
+What is the network name for Incentivized Testnet?
+--------------------------------------------------
+
+The network name is ``cassandra``.
+
+
+How long is a period?
+---------------------
+
+1 period equals 24 hours. Periods begin at midnight UTC.
+
+
+What's the difference between staker address and worker address? Can they be the same?
+--------------------------------------------------------------------------------------
+
+Technically, yes, but it is not recommended. The accounts have different security considerations - the staker address
+can be a high-value hardware wallet (with NU and ETH) that performs stake management while the worker
```

Comment:
Some participants – in CASI and on mainnet – won't own high-value hardware wallets. I assume we don't want to limit our staker pool to those that do. So this FAQ might prompt another: 'what is the recommended set-up for my NU stake if I don't have the capital and/or expertise to run a YubiHSM?'